community/bolt/transport-socket/pom.xml

```diff
@@ -50,6 +50,15 @@
     </dependency>
 
     <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk15on</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
```

Comment:
Hmm. I'll write a PR right now then to remove Bolt.

On Mon, Nov 2, 2015 at 2:55 PM, Konstantin Lutovich <
notifications@github.com> wrote:

> In community/bolt/transport-socket/pom.xml
> <https://github.com/neo4j/neo4j/pull/5812#discussion_r43680711>:
>
> > @@ -50,6 +50,15 @@
> >      </dependency>
> >
> >      <dependency>
> > +      <groupId>org.bouncycastle</groupId>
> > +      <artifactId>bcprov-jdk15on</artifactId>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.bouncycastle</groupId>
> > +      <artifactId>bcpkix-jdk15on</artifactId>
>
> Bolt tests are still executed during the build. And without bouncycastle
> some tests fail with because neither sun.security.x509 nor
> org.bouncycastle.cert are in the classpath.
> If Bolt is going to be removed than it is better to wait with this PR and
> change only server pom.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neo4j/neo4j/pull/5812/files#r43680711>.
>